README.md

```diff
@@ -220,10 +220,10 @@ Common configurations:
 |field|default value|description|
 |-|-|-|
 |mongodb.uri|"mongodb://localhost:27017"|Connection string URI format|
-|mongodb.ns|"ycsb.ycsb"|Namespace|
-|mongodb.user|"root"|Database Username|
-|mongodb.passw|""|Database Password|
-|mongodb.dbauth|"admin"|Admin Database|
+|mongodb.namespace|"ycsb.ycsb"|Namespace to use|
+|mongodb.authenticationDatabase|"admin"|Authentication database|
```

Comment:
Is "authdb" ok?

BTW:
The name "authenticationDatabase" refers to "mongo" command options.
"mongo --help" output like this:
```
Authentication Options:
  -u [ --username ] arg               username for authentication
  -p [ --password ] arg               password for authentication
  --authenticationDatabase arg        user source (defaults to dbname)
  --authenticationMechanism arg       authentication mechanism
  --gssapiServiceName arg (=mongodb)  Service name to use when authenticating 
                                      using GSSAPI/Kerberos
  --gssapiHostName arg                Remote host name to use for purpose of 
                                      GSSAPI/Kerberos authentication
```